salesforce/functions.py

```diff
@@ -0,0 +1,13 @@
+from .salesforce import Salesforce
+
+
+def retrieve_salesforce_abbreviation(sf_book_name):
+    if sf_book_name:
+        with Salesforce() as sf:
+            command = "Select Name from Book__c where Official_Name__c = '" + sf_book_name + "'"
```

Comment:
I'm wondering, instead of the name - should we use the Salesforce ID? That way we are sure the book exists and is purposely connected to that record?

Reply:
I don't have access to the sf book id in the book model

Reply:
I was thinking that is what content managers would input into the CMS to connect the book to Salesforce (part of a new creation, is making sure the book is also created in Salesforce - so this would help enforce/remind about that).

Reply:
Just so I understand, the content managers enter the book id and I look up the 2 name fields on save. The name fields are not editable and the book id is not in the API.